### PR TITLE
add telemetry to BSI

### PR DESCRIPTION
**Pull request by danyakarras**
Not sure exactly why there are merge conflicts in the package-lock.json, and whether I should be pushing that file, or how to resolve them. Please let me know!

### CODE REVIEW

**Comment by dlockhart** [comment]
There's some [documentation in the README](https://github.com/Brightspace/brightspace-integration#npm-dependency-locking) on how to deal with the `package-lock.json` file. See if following those steps fixes things.

File: package-lock.json [hunk 1]
```diff
@@ -3054,12 +3054,22 @@
...
-      "integrity": "sha512-qkuP/vSrRNe/ccDwPTH8T6tgbkpVpTgFUVgaaBReXhl+byQKiDy8LJ2OqaumktYTwvpUObDTCz6W2F1q0agdxw==",
+      "version": "1.5.0",
+      "resolved": "https://registry.npmjs.org/d2l-fetch-auth/-/d2l-fetch-auth-1.5.0.tgz",
+      "integrity": "sha512-JdnoI8UOso9jTlFXlhRRcTVzgFw/vnOxwzc3A/2E5JgEh9taqmqAP46CQTf6a/FFgmZJhpaRWMBB0n2VBRPXWA==",
+      "dev": true,
+      "requires": {
+        "d2l-fetch-auth-trust": "github:Brightspace/d2l-fetch-auth-trust#semver:^1",
+        "frau-jwt": "^2.0.3"
+      }
+    },
```
Comment (dlockhart) [inline package-lock.json]: FYI @omsmith

File: package-lock.json [hunk 2]
```diff
@@ -3320,7 +3330,7 @@
       }
     },
     "d2l-navigation": {
-      "version": "github:BrightspaceUI/navigation#cabd74575679dd3c88b7e1468e9ea80d248f38ad",
+      "version": "github:BrightspaceUI/navigation#169e2220fa992e1292e4e8640cf12563571c2b5d",
```
Comment (dlockhart) [inline package-lock.json]: FYI @jeroach

File: package-lock.json [hunk 3]
```diff
@@ -3366,7 +3377,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#187888f860ab8e2829633ddbf7523cb11fe407ef",
+      "version": "github:BrightspaceHypermediaComponents/organizations#23a653dc62c5a19e1298329eba50b83edd3b9aac",
```
Comment (dlockhart) [inline package-lock.json]: FYI @ciss1995

File: package-lock.json [hunk 4]
```diff
@@ -3431,7 +3443,7 @@
       }
     },
     "d2l-polymer-siren-behaviors": {
-      "version": "github:Brightspace/polymer-siren-behaviors#0a3c49aca4cd66a3b4c6c6f47a36edbbb9bb0b69",
+      "version": "github:Brightspace/polymer-siren-behaviors#9120d2deee9b4fc8b31a9e436f130357cd2ac6ed",
```
Comment (dlockhart) [inline package-lock.json]: FYI @omsmith @awikkerink @pbrown-d2l

File: package-lock.json [hunk 5]
```diff
@@ -3546,7 +3558,7 @@
       }
     },
     "d2l-sequence-viewer": {
-      "version": "github:Brightspace/d2l-sequence-viewer#9a8ae838c5d08689b10147abf89f0108b636ee43",
+      "version": "github:Brightspace/d2l-sequence-viewer#e4f18abc84e4f021ca990ec535318a7dea78a016",
```
Comment (dlockhart) [inline package-lock.json]: FYI @ChrisKraljevicD2L

File: package-lock.json (hunk 2 above)
Comment (jeroach) [reply package-lock.json]: Oh cool, was going to bump myself :)

Thanks!

File: package-lock.json (hunk 4 above)
Comment (omsmith) [reply package-lock.json]: Could intentionally (as per the PR) cause issues. Keep an eye out for `Invalid request url; must be a valid whitelisted domain.`

**Pull request merged**